static/src/javascripts/projects/common/modules/identity/ad-prefs/ConsentBox.js

```diff
@@ -67,8 +67,11 @@ class ConsentBox extends Component<ConsentBoxProps, {}> {
         return (
             <fieldset>
                 <legend className="identity-title identity-title--small">
-                    {wording.question}
+                    {wording.question.title}
                 </legend>
+                <div className="identity-ad-prefs-input__wording">
+                    {wording.question.text}
+                </div>
```

Comment:
Since `wording.question.text ` is optional ideally we'd need to be wrapped like this to not show an empty div if it's not there

```suggestion
                {wording.question.text &&
                    <div className="identity-ad-prefs-input__wording">
                        {wording.question.text}
                    </div>
                }
```

Reply:
Thanks Laura. Have amended that now.